examples/06_Combining_Models.ipynb

```diff
@@ -6,6 +6,250 @@
    "source": [
     "# Combining Prognostic Models"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This section demonstrates how prognostic models can be combined. There are two times in which this is useful: \n",
+    "\n",
+    "1. When combining multiple models of different inter-related systems into one system-of-system model (i.e., [Composite Models](https://nasa.github.io/progpy/api_ref/prog_models/CompositeModel.html)), or\n",
+    "2. Combining multiple models of the same system to be simulated together an aggregated (i.e., [Ensemble Models](https://nasa.github.io/progpy/api_ref/prog_models/EnsembleModel.html)). This is generally done to improve the accuracy of prediction when you have multiple models that each represent part of the behavior or represent a distribution of different behaviors. \n",
+    "\n",
+    "These two methods for combining models are described in the following sections."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Composite Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A CompositeModel is a PrognosticsModel that is composed of multiple PrognosticsModels. This is a tool for modeling system-of-systems. i.e., interconnected systems, where the behavior and state of one system effects the state of another system. The composite prognostics models are connected using defined connections between the output or state of one model, and the input of another model. The resulting CompositeModel behaves as a single model.\n",
+    "\n",
+    "To illustrate this, we will create a compositemodel of an aircraft's electric powertrain, combining the DCMotor, ESC, and PropellerLoad models.\n",
```

Comment:
```suggestion
    "To illustrate this, we will create a composite model of an aircraft's electric powertrain, combining the DCMotor, ESC, and PropellerLoad models.\n",
```

Reply:
Do you think it would be worthwhile to add just a sentence or two about how these three models work together to create an electric powertrain? Something short, and then may add a reference to the documentation or the notebook that describes them? 

I'm thinking that may help the concept of composite models to be clear for someone who may not know much about electric powertrains.